subjects/java/projects/travel-plan/audit/README.md

```diff
@@ -0,0 +1,80 @@
+
+#### Comprehension
+
+##### Ask the student to describe the microservices architecture implemented.
+
+###### Did the architecture effectively split functionalities into different microservices based on key factors like scalability and domain separation?
+
+##### Ask the student to explain one of the Ansible playbook
+
+###### did he/she clearly explain all the Ansible playbook
+
+##### Discuss the CI/CD pipeline setup.
+
+###### is there unit tests for each functionality and are the test run for each PR.
+
+###### is SonarQube report free from any error or warning that can break the CI/CD Process.
+
+##### Detail the security measures implemented.
+
+###### Were comprehensive security measures like SSL/TLS, secret management, and principle of least privilege correctly implemented?
+
+##### Ask the student to explain the database schema for PostgreSQL and Neo4j.
+
+###### Did the data structure in PostgreSQL and Neo4j effectively support the application's requirements?
+
+#### Functional
+
+##### Verify the execution of Ansible playbooks.
+
+###### Did the Ansible playbooks execute without errors and configure the environment as intended?
+
+###### Were the playbooks able to handle re-running scenarios without causing disruptions or inconsistencies?
+
+##### Test each microservice API.
+
+###### Are all microservice APIs is role based and doesn't work with a admin role based user authenticated except the login api?
```

Comment:
This point is not clear - is this supposed to be something like:
```
Are all API endpoints protected by authentication? Are all the endpoints only accessible to Admin users? (This last question might be not relevant for this project - unless the login/registration features are required for users as well. If this is the case, please, make it clearer in the subject)

Reply:
when we ask them to implement authentication and authorization this include that they have to build a login page !!
okay i will apply the  change that you suggest looks better. thanks

Reply:
This sentence is hard to understand for me. A possible rephrasing (but it might not be what you mentioned in the first place)

```
Is there a role-based management authentication implemented to access all the microservices' APIs? 
```

It is never mentioned in the subject to implement such a thing. 

Another option could be:
``
Are all the microservices' APIs only accessible when logged in with an Admin profile?
```

Or should be there some API (that displays data for example) available in general?